Chapter_5/emr/README.md

```diff
@@ -1,55 +1,75 @@
-### Setting up an EMR Cluster for ETL
+### EMR Setup
```

Comment:
Can you keep the heading as is.

I am trying to align them with the headings in the book.

`### Setting up an EMR Cluster for ETL`

Reply:
done